### PR TITLE
13049 Add DEIS attachment

### PR DESCRIPTION
**Pull request by CeciliaCY**
### Summary
13049 Add Attachments section to DEIS

#### Task/Bug Number
[Fix AB 13049](https://dcp-paperless.visualstudio.com/ZAP%20Portals/_workitems/edit/13049)

### Technical Explanation
Same as Draft EAS

### CODE REVIEW

**Review by allthesignals — COMMENTED** [review]
@godfreyyeung @trbmcginnis is there an acceptnace test for this section? could we get some boilerplate test in there at some point? 

@CeciliaCY does this work (are you abble to attach docs to DEIS?)? what's left? backend stuff?

File: client/app/components/packages/deis/attached-documents.hbs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  <Packages::Attachments
+    @package={{@model}}
+    @fileManager={{@model.fileManager}}
+    data-test-section="attachments"
+  />
+</@form.Section>
+
+
+
```
Comment (allthesignals) [inline client/app/components/packages/deis/attached-documents.hbs:17]: tiny nit but why all this extra space? should always only be 1 carriage return after the last non-empty character.

**Comment by godfreyyeung** [comment]
> @godfreyyeung @trbmcginnis is there an acceptnace test for this section? could we get some boilerplate test in there at some point?
> 
> @CeciliaCY does this work (are you abble to attach docs to DEIS?)? what's left? backend stuff?

Good callout. I'm going to take some time to write some tests

File: client/app/components/packages/deis/attached-documents.hbs (hunk 1 above)
Comment (CeciliaCY) [reply client/app/components/packages/deis/attached-documents.hbs:17]: Not sure. Is there a good way to check/valid these in VS code while coding?

**Comment by CeciliaCY** [comment]
@trbmcginnis yes, it's working. I'm able to upload the files.

**Review by godfreyyeung — APPROVED** [review]

**Pull request merged**